platforms/chibios/drivers/spi_master.c

```diff
@@ -18,12 +18,20 @@
 
 #include "timer.h"
 
-static pin_t currentSlavePin = NO_PIN;
+static bool spiStarted = false;
 
-#if defined(K20x) || defined(KL2x) || defined(RP2040)
+#if SPI_SELECT_MODE != SPI_SELECT_MODE_NONE
+#    if defined(K20x) || defined(KL2x) || defined(RP2040)
 static SPIConfig spiConfig = {NULL, 0, 0, 0};
-#else
+#    else
 static SPIConfig spiConfig = {false, NULL, 0, 0, 0, 0};
+#    endif
+#else
+#    if defined(K20x) || defined(KL2x) || defined(RP2040)
+static SPIConfig spiConfig = {NULL, 0};
+#    else
+static SPIConfig spiConfig = {false, NULL, 0, 0};
+#    endif
 #endif
```

Comment:
Can we just rely on the default zero initialization for `static` variables here?
```c
static SPIConfig spiConfig;
```
(GitHub does not allow adding a proper suggestion for this code block.)

Reply:
DONE